src/libstd/sys/common/condvar.rs

```diff
@@ -18,14 +20,14 @@ use sys::condvar as imp;
 /// condition variables. It is consequently entirely unsafe to use. It is
 /// recommended to use the safer types at the top level of this crate instead of
 /// this type.
-pub struct Condvar(imp::Condvar);
+pub struct Condvar(Box<imp::Condvar>);
```

Comment:
@retep998 This PR does not add any additional boxes. It just has been moved from sync::Condvar::inner.